tests/conftest.py

```diff
@@ -123,7 +123,14 @@ def destroy(self):
             ensure_system("ip netns delete %s" % self.nsname)
 
     def runcmd(self, cmd):
-        return os.system("ip netns exec %s %s" % (self.nsname, cmd))
+        try:
+            out = subprocess.check_output("ip netns exec %s %s" % (self.nsname, cmd), stderr=subprocess.STDOUT, shell=True)
+        except subprocess.CalledProcessError as e:
+            print "------rc={} for cmd: {}------".format(e.returncode, e.cmd)
+            print e.output
```

Comment:
the error message will always follow with \n, so there will be an extra empty line when using print function

Reply:
resolved.